scripts/mas_sso.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+docker run -d -p 8180:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin \
```

Comment:
let's also configure this as GH actions services: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/11c964a74facabc3d7c1a121e3984ffd3eb3c53b/.github/workflows/ci.yaml#L70